src/core/lib/transport/parsed_metadata.h

```diff
@@ -147,7 +147,7 @@ class ParsedMetadata {
     grpc_error_handle (*const set)(const Buffer& value,
                                    MetadataContainer* container);
     ParsedMetadata (*const with_new_value)(const Buffer& value,
```

Comment:
Maybe add a comment here explaining that the `new_value` param needs to be a pointer instead of being passed by value to work around the gcc 4.9 bug?  That way, we will know we can change it once we no longer support that compiler.